pages/agent/v3/tokens.md

```diff
@@ -1,100 +1,126 @@
 
-## The default token
+## The initial agent token
 
-When you create a new organization in Buildkite, a default agent token is created. This token can be used for testing and development, but it's recommended you [create new, specific tokens](#creating-tokens) for each new environment.
+When you create a new organization in Buildkite, an initial agent token is created (called _Initial agent token_ within the _Default cluster_). This token can be used for testing and development and is only revealed once during the organization setup process. However, it's recommended that you [create new, specific tokens](#create-a-token) for each new environment.
 
 ## Using and storing tokens
 
-The token is used by the Buildkite Agent's [start](/docs/agent/v3/cli-start#starting-an-agent) command, and can be provided on the command line, set in the [configuration file](/docs/agent/v3/configuration), or provided using the [environment variable](/docs/pipelines/environment-variables) `BUILDKITE_AGENT_TOKEN`.
+An agent token is used by the Buildkite Agent's [start](/docs/agent/v3/cli-start#starting-an-agent) command, and can be provided on the command line, set in the [configuration file](/docs/agent/v3/configuration), or provided using the [environment variable](/docs/pipelines/environment-variables) `BUILDKITE_AGENT_TOKEN`.
 
 It's recommended you use your platform's secret storage (such as the [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html)) to allow for easier rollover and management of your agent tokens.
 
-## Creating tokens
-
-New tokens can be created using the [GraphQL API](/docs/apis/graphql-api) with the `agentTokenCreate` mutation.
-
-For example:
-
-```graphql
-mutation {
-  agentTokenCreate(input: {
-    organizationID: "organization-id",
-    description: "A description"
-  }) {
-    tokenValue
-    agentTokenEdge {
-      node {
-        id
-      }
-    }
-  }
-}
-```
+## Create a token
+
+New agent tokens can be created either using the [_Agent Tokens_ page of a cluster](#create-a-token-using-the-buildkite-interface), or via the [REST API's create agent token](#create-a-token-using-the-rest-api) feature.
+
+> 📘 An agent token's value is only displayed once
+> As soon as the agent token's value is displayed, copy its value and save it in a secure location.
+> If you forget to do this, you'll need to create a new token to obtain its value.
+
+It is possible to create multiple agent tokens (for your Default cluster or any other cluster in your Buildkite organization) using the processes described in this section.
+
+### Using the Buildkite interface
+
+To create an agent token for a cluster using the Buildkite interface:
+
+1. Select _Agents_ to access the _Agent Clusters_ page.
+1. Select the cluster containing the agent token to revoke.
```

Comment:
```suggestion
1. Select the cluster you would like to create an agent token for.
```

Reply:
@gilesgas I think you'll be able to come up with better words here - the main point is that I think 'revoke' snuck through when it should have been a 'create' in this section.

Reply:
Whoops! Copy-and-paste error 🤦